src/dagmc.cpp

```diff
@@ -609,14 +631,38 @@ void DAGUniverse::uwuw_assign_material(
   fatal_error("DAGMC was not configured with UWUW.");
 #endif // OPENMC_UWUW
 }
+
+void DAGUniverse::override_assign_material(std::string key,
+  moab::EntityHandle vol_handle, std::unique_ptr<DAGCell>& c) const
+{
+
+  // if Cell ID matches an override key, use it to override the material
+  // assignment else if UWUW is used, get the material assignment from the DAGMC
+  // metadata
+  std::cout << "XML instance"
+            << instance_material_overrides.count("id_" + std::to_string(c->id_))
+            << std::endl;
```

Comment:
We probably don't want all of this output every time. Let's use the existing `message` functions and select an appropriate verbosity level for displaying this information.